Dockerfile

```diff
@@ -16,6 +16,7 @@ RUN yarn install
 
 ENV PROVIDER PROVIDER
 ADD contracts contracts
+ADD tests tests
```

Comment:
Do you actually run tests in Docker? I guess that's pretty cool for isolation

Reply:
I had to add that for saddle config to find ProxyPriceOracle.sol in tests/contracts for mocking anchor oracle in tests.